desktop/core/src/desktop/js/apps/storageBrowser/StorageFilePage/StorageFilePage.util.ts

```diff
@@ -53,8 +57,16 @@ export const getFileMetaData = (t: TFunction, fileStats: FileStats): MetaData[][
       {
         name: 'mtime',
         label: t('Last Modified'),
-        value: fileStats.mtime ? formatTimestamp(new Date(fileStats.mtime * 1000)) : '-'
+        value: formatTimestamp(new Date(fileStats.mtime * 1000))
       }
     ]
   ];
 };
+
+export const getFileType = (fileName: string): SupportedFileTypes => {
+  const fileExtension = fileName?.split('.')?.pop()?.toLocaleLowerCase();
```

Comment:
Nit: Could there be trouble with different locales here? Perhaps safer to go for toLowerCase()..